### PR TITLE
Multiple feature additions

### PR DESCRIPTION
**Pull request by someshkoli**
1. add workflow link to comment (#14)
2. Add header and footer to the comment
3. quite mode. dont comment if there are no changes (#11)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -113,13 +126,21 @@ try {
         process.exit(0);
     }
 
+    if (quiteMode && hasNoChanges) {
```
Comment (RobertKelly) [inline index.js:129]:
```suggestion
    if (quietMode && hasNoChanges) {
```

File: index.js [hunk 2]
```diff
@@ -113,13 +126,21 @@ try {
...
+        core.info("Quite mode is enabled and there are no changes to the infrastructure.")
```
Comment (RobertKelly) [inline index.js:130]:
```suggestion
        core.info("Quiet mode is enabled and there are no changes to the infrastructure.")
```

File: .github/workflows/action-test.yml [hunk 3]
```diff
@@ -26,10 +26,14 @@ jobs:
...
         with:
           json-file: test-data/tf_test.json
-          expand-comment: 'true'
-          include-plan-job-summary: 'true'
+          expand-comment: "true"
+          include-plan-job-summary: "true"
+          comment-header: "BIG HEADER"
+          comment-footer: "BIG FOOTER"
+          include-workflow-link: "true"
+          quite: "true"
```
Comment (RobertKelly) [inline .github/workflows/action-test.yml:34]:
```suggestion
          quiet: "true"
```

File: index.js [hunk 4]
```diff
@@ -8,26 +8,37 @@ const myToken = core.getInput('github-token');
 const octokit = github.getOctokit(myToken);
 const context = github.context;
 const inputFilenames = core.getMultilineInput('json-file');
+const commentHeader = core.getMultilineInput('comment-header');
+const commentFooter = core.getMultilineInput('comment-footer');
+const quiteMode = core.getMultilineInput('quite');
```
Comment (RobertKelly) [inline index.js:13]:
```suggestion
const quietMode = core.getMultilineInput('quiet');
```

**Review by RobertKelly — CHANGES_REQUESTED** [review]
Made some suggestions: `quite` to `quiet`

**Comment by someshkoli** [comment]
This pr has become absolute mess in attempt to get things working ☠️
Will raise a new one after after some cleanup

**Comment by someshkoli** [comment]
Closing in prefrence to #60

**Pull request closed without merging**